sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -59,6 +60,12 @@ private ProjectBuilder AddProject(AnalyzerLanguage language, string projectName,
 
             var project = Solution.AddProject(projectName, projectName, languageName);
 
+            var compilationOptions = project.CompilationOptions.WithOutputKind(OutputKind.DynamicallyLinkedLibrary);
+            if (languageName == LanguageNames.CSharp)
```

Comment:
Shouldn't we handle also VB.Net?

Reply:
_Joke:_ Because it's always safe to use VB.NET 😆 

There's not `unsafe` code in VB, the option doesn't exist.